examples/jdbc-query-operation/jdbc_query_operation.bal

```diff
@@ -1,5 +1,5 @@
 import ballerina/io;
-import ballerina/jdbc;
+import ballerina/java.jdbc as jdbc;
```

Comment:
```suggestion
import ballerina/java.jdbc;
```
This we can simply leave it this, can't we? In this case also the alias will be `jdbc`